sky/jobs/utils.py

```diff
@@ -69,8 +71,10 @@
 # The maximum time to wait for the managed job status to transition to terminal
 # state, after the job finished. This is a safeguard to avoid the case where
 # the managed job status fails to be updated and keep the `sky jobs logs`
-# blocking for a long time.
-_FINAL_JOB_STATUS_WAIT_TIMEOUT_SECONDS = 25
+# blocking for a long time. This should be significantly longer than the
+# JOB_STATUS_CHECK_GAP_SECONDS to avoid timing out before the controller can
+# update the state.
+_FINAL_JOB_STATUS_WAIT_TIMEOUT_SECONDS = 40
```

Comment:
Does this cause a `sky jobs launch echo hi` taking longer to finish the logs at the end and return?

Reply:
I see! The end-to-end `sky jobs launch echo hi` would be metrics for the performance of our managed jobs, but we can worry about it later. :)